lib/Module2Rpm/Deb.rakumod

```diff
@@ -0,0 +1,146 @@
+    }
+
+    #| Returns a list of the required modules with the pattern:
+    #| Requires:       <name of the requirement>
+    method requires() {
+        (
+            '${misc:Depends}',
+            '${raku:Depends}',
+            'rakudo (>= 2022.07-2)',
+            |$!metadata.requires.map({ self.map-dependency($_) })
+        )
+    }
+
+    method test-requires() {
+        $!metadata.test-requires.map({ self.map-dependency($_) })
+    }
+
+    #| Returns a list of the build requirements with the pattern:
+    #| BuildRequires:  <name of the build requirement>
+    method build-requires()  {
+        (
+            'debhelper-compat (= 13)',
+            'dh-sequence-raku',
+            'rakudo (>= 2022.07-2)',
+            |$!metadata.build-requires.map({ self.map-dependency($_) }),
+            |$!metadata.test-requires.map({ self.map-dependency($_) }),
+        )
+    }
+
+    method map-dependency(Module2Rpm::Requires $requires)  {
+        given $requires.adverbs<from> {
+            when 'native' {
+                $!log.debug("Deb.map-dependency: Look for native library name: $requires.name()");
+                return "lib$requires.name()"; #TODO actually look for matching package
+            }
+            when 'bin'    {
+                if $requires.name eq 'perl' {
+                    $!log.debug("Bin requires: perl");
+                    return "perl";
+                }
+
+                my $req = $requires.name; #TODO actually look for matching package
+                $!log.debug("Bin requires: $req");
+
+                return $req;
+            }
+            default       {
+                my $req = "raku-{$requires.name.lc.subst('::', '-', :g)}";
+                $!log.debug("Default requires: $req");
+                return $req;
+            }
+        }
+    }
+    method write-build-files(IO::Path $path, :$build-file) {
+        if $build-file {
+            my $rules = %?RESOURCES<debian.rules.with-build>.IO;
+            $path.add('debian.rules').spurt: render-template($rules, {file => $build-file.basename});
+        }
+        else {
+            %?RESOURCES<debian.rules>.copy($path.add('debian.rules'), :createonly);
+        }
+
+        my $package-name = self.metadata.get-package-name.lc;
+        my $version = S/^v// given $!metadata.get-version();
+        my $timestamp = DateTime.now(formatter => {
+            sprintf "%s, %d %s %d %02d:%02d:%02d +%02d00",
+                <Mon Tue Wed Thu Fri Sat Sun>[.day-of-week - 1],
+                .day,
+                <Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec>[.month - 1],
+                .year,
+                .hour,
+                .minute,
+                .second,
+                .offset-in-hours
+        });
+
+        $path.add('debian.control').spurt: qq:to/END/;
+            Source: $package-name
+            Maintainer: Stefan Seifert <stefan.seifert@rootprompt.at>
```

Comment:
This module will be published with fez to the raku ecosystem. Hard coded values does not look like a good idea. There is a config file used for the credentials. Maybe also the place for these?